.github/workflows/gradle-release.yml

```
@@ -0,0 +1,38 @@
+
+    - name: Git Config
+      run: |
+        git config user.name 'Nya Ξlimu'
+        git config user.email 'info@elimu.ai'
+
+    - name: Remove -SNAPSHOT from versionName
+      run: ./gradlew removeSnapshot
+
+    - name: Set the new versionName as an environment variable
+      id: version_name
+      run: |
```

actionlint (1.7.4)

undefined variable "VERSION_NAME". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)